pkg/tokens/builtin/issuer/issuer.go

```diff
@@ -5,9 +5,10 @@ import (
 	"github.com/pkg/errors"
 
 	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
-	"github.com/kumahq/kuma/pkg/sds/auth"
 )
 
+type Token = string
```

Comment:
introduced token type, otherwise we were spreading SDS dependency to other parts fo the code.